Package.swift

```diff
@@ -2,7 +2,7 @@
 import PackageDescription
 
 let package = Package(
-  name: "CasePaths",
+  name: "case-paths",
```

Comment:
Should this be `swift-case-paths` per https://github.com/apple/swift-standard-library-preview/blob/833ea174a771bdc98edfa3529ae05439564e3902/Package.swift#L43?

Reply:
I wondered that too, but I decided to leave that decision up to you.